query/autocomplete.js

```diff
@@ -13,7 +13,8 @@ query.score( peliasQuery.view.ngrams, 'must' );
 
 // scoring boost
 query.score( peliasQuery.view.phrase );
-query.score( peliasQuery.view.focus );
+query.score( peliasQuery.view.focus( peliasQuery.view.ngrams ) );
+query.score( peliasQuery.view.popularity(['admin0','admin1','admin2']) );
```

Comment:
this is actually really cool to see how this one line transforms into a whole nice big section of the query